client/me/purchases/purchases-list-in-dataviews/purchases-data-view.ts

```diff
@@ -0,0 +1,17 @@
+export const purchasesDataView = {
+	type: 'table',
+	search: [ 'blog_id', 'product_id' ],
```

Comment:
The search and filters are left over from initial proof of concepts. They need to be updated to work dynamically with the API data. I just haven't gotten there yet.